Headless portal (#2082)
umbraco/UmbracoDocs: Umbraco-Headless/Getting-Started-Cloud/The-Headless-Portal/index.md

# The Umbraco Heartcore Portal

> 

Reviewer: Should we add some intro text here, explaining what kind of information can be found in the article?

> After you have logged into the Cloud Portal you are presented with the project overview.

Reviewer: Would be great to have a link to the Portal from here!

![Portal Project Overview](images/portalOverview.png)

From here you can see a list of all your projects. If you have many projects, you are able to search for specific projects from the search bar in the top right corner.

Clicking on a project will take you to the project where you will get an overview of your environment(s).

![Portal Project Overview](images/projectOverview.png)

## Manage Environments

> From here you are able to create multiple environment so you might have a Development environment as well. Depending on what plan you have chosen, you will have acces to one or more environment on your project.

Reviewer: ```suggestion
From here you are able to create multiple environments so you might have a Development environment as well. Depending on what plan you have chosen, you will have acces to one or more environment on your project.
```

![Manage the environments](images/manageEnvironments.png)

If you want to add an environment all you have to do is click the big green button. Umbraco will create the environment for you.

> When you are done you can go a head and click on the Done, take me back link in the top-right corner to get back to the project overview.

Reviewer: ```suggestion
When you are done you can go ahead and click on 'Done, take me back' in the top-right corner to get back to the project overview.
```

![Invite Team Member Modal](images/inviteModal.png)

## Settings

Right next to the invite member option. you will find the **Settings** dropdown menu. Here you will find various links to more management and configuration options.

### Edit Team

As with the **invite member** option you are able to add new team members to your project. You will also be able to remove team members from this page as well as see any pending invitations you might have sent.

It is also here you are able to add the Technical Contact information such as Name, Email and a telephone number.

![Edit Team Members page](images/editTeam.png)

### Webhooks

Here you are able to set up a webhook that will gather all the information related to deployments. This information is very useful if you har set up something like Slack integration so every time a deployment has been made a message will be posted.

> The webhooks that can be added here is not the same as the webhooks that can be created from the Backoffice.

Reviewer: Would be cool if this was added like a note or tip instead, right?

Check out this article to learn more about how to format notes and tips: https://our.umbraco.com/documentation/Contribute/Markdown-Conventions/#adding-notes-warnings-tips

### Upgrade plan

Here you will be able to upgrade your trial to a plan that fits your needs.

### Rename project

> Here you are able to rename your project. All default umbraco.io hostnames will be updated to match the new name. Also note that all clone urls will be updated to match the new projectname. If you have local git repositories remember to update the urls.

Reviewer: I don't think we should mention "cloning down" and "local repositories" here, as that will not be an option for Heartcore. What do you think?

```suggestion
Here you are able to rename your project. All default umbraco.io hostnames will be updated to match the new name. Also note that all clone URLs will be updated to match the new projectname. If you have local git repositories remember to update the URLs.
```

Reviewer 2: Maybe instead we could mention that you'll need to update the `Umb-Project-Alias` header in the requests to the APIs?

![Rename Project Page](images/renameProject.png)

### Delete project

From this page you can delete your project. When deleting the project your subscription will automatically be cancelled as well. Deleting the project is a **permanent action**.

![Delete project](images/deleteProject.png)

## Go to backoffice

> Clicking on the Go to backoffice link will open up a new tab with the login screen for the Backoffice. If you would like to learn more, you can read our documentation for the [Heartcore Backoffice](../The-Headless-Portal).

Reviewer: ```suggestion
Clicking on the 'Go to backoffice' link will open up a new tab with the login screen for the Backoffice. If you would like to learn more, you can read our documentation for the [Heartcore Backoffice](../The-Headless-Portal).
```

![Invite Team Member Modal](images/inviteModal.png)

## Settings

Right next to the invite member option. you will find the **Settings** dropdown menu. Here you will find various links to more management and configuration options.

### Edit Team

As with the **invite member** option you are able to add new team members to your project. It is also possible to change the role for the team member here. You will also be able to remove team members from this page as well as see any pending invitations you might have sent.

It is also here you are able to add or edit the Technical Contact information such as Name, Email and a telephone number.

![Edit Team Members page](images/editTeam.png)

### Webhooks

Here you are able to set up a webhook that will gather all the information related to deployments. This information is very useful if you har set up something like Slack integration so every time a deployment has been made a message will be posted.

:::note
The webhooks that can be added here is **not the same** as the webhooks that can be created from the Backoffice.
:::

### Upgrade plan

Here you will be able to upgrade your trial to a plan that fits your needs.

### Rename project

> Here you are able to rename your project. All default umbraco.io hostnames will be updated to match the new name. When renaming a project you will also need to update the `Umb-Project-Alias` header when you are sending requests to the API.

Reviewer: ```suggestion
Here you are able to rename your project. All default umbraco.io hostnames will be updated to match the new name. When renaming a project you will also need to update the `Umb-Project-Alias` header when you are sending requests to the API, as changing the name of the project will also change the project alias.
```
